src/bo4e/enum/messwertstatus.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=missing-module-docstring
+from bo4e.enum.strenum import StrEnum
+
+
+class Messwertstatus(StrEnum):
+    """
+    Der Status eines Zählerstandes
+    """
+
+    ABGELESEN = "ABGELESEN"  #: ABGELESEN
+    ERSATZWERT = "ERSATZWERT"  #: ERSATZWERT
+    VORSCHLAGSWERT = "VORSCHLAGSWERT"  #: VORSCHLAGSWERT
+    NICHT_VERWENDBAR = "NICHT_VERWENDBAR"  #: NICHT_VERWENDBAR
+    PROGNOSEWERT = "PROGNOSEWERT"  #: PROGNOSEWERT
+    VOLAEUFIGERWERT = "VOLAEUFIGERWERT"  #: VOLAEUFIGERWERT
```

Comment:
I think this is a typo in the BO4E docu
```suggestion
    VORLAEUFIGERWERT = "VORLAEUFIGERWERT"  #: VORLAEUFIGERWERT
```

Reply:
danke. war wohl vorher auch nie sauber reviewed worden